modules/conversion-center/base/widget-link-in-bio-base.php

```diff
@@ -796,13 +796,7 @@ protected function add_advanced_tab(): void {
 			],
 		);
 
-		// Getting active breakpoints and setting dynamic options
-		$active_breakpoints = Plugin::$instance->breakpoints->get_active_breakpoints();
-
-		foreach ( $active_breakpoints as $breakpoint_key => $breakpoint ) {
-			$available_devices[ $breakpoint_key ] = $breakpoint->get_label();
-			$default_devices[] = $breakpoint_key;
-		}
+		$configured_breakpoints = $this->get_configured_breakpoints();
```

Comment:
@willhowat Can you not use `Plugin::$instance->breakpoints->get_active_devices_list()`?

Reply:
@louiswol94 Done